lib/babel.js

```diff
@@ -2,6 +2,8 @@
 var resolveFrom = require('resolve-from');
 var createEspowerPlugin = require('babel-plugin-espower/create');
 var requireFromString = require('require-from-string');
+var hasha = require('hasha');
+var cache = require('cacha')('.ava/cache');
```

Comment:
Can you make sure it follow XDG?

https://github.com/sindresorhus/xdg-basedir#cache